boringtun/src/noise/mod.rs

```diff
@@ -438,6 +441,10 @@ impl TunnInner {
 
         let (packet, session) = self.handshake.receive_handshake_initialization(p, dst)?;
 
+        // We received a valid handshake initialization
+        // Increase the rx_bytes accordingly
+        self.rx_bytes += HANDSHAKE_INIT_SZ;
```

Comment:
I looked over code more, and think that this approach is good :+1:

Reply:
I tried to keep the changes as small as possible. Here, the packet.len() is actually the size of the handshake response message and it's used a bit lower to increase tx_bytes. 
The parsing of the handshake init message happens earlyer and it's harder to track the bytes using something like packet.len(). In my opinion it doesn't make sense to add extra logic for that, because here we know for sure we just received a valid handshake initialization message.